analyzers/tests/SonarAnalyzer.UnitTest/TestFramework/Verifier.cs

```diff
@@ -298,16 +405,60 @@ static IEnumerable<TMessage> ReadProtobuf(string path)
                                            OutputKind outputKind,
                                            IEnumerable<MetadataReference> additionalReferences,
                                            string sonarProjectConfigPath = null)
+        {
+            using var scope = new EnvironmentVariableScope(true) { EnableConcurrentAnalysis = true};
+            CreateConcurrencyTest(paths, out var pathsWithConcurrencyTests);
+            var solution = SolutionBuilder.CreateSolutionFromPaths(pathsWithConcurrencyTests, outputKind, additionalReferences);
+            CompileAndVerifyAnalyzer(solution, diagnosticAnalyzers, options, checkMode, sonarProjectConfigPath);
+        }
+
+        private static void CreateConcurrencyTest(IEnumerable<string> paths, out List<string> resultingPaths)
```

Comment:
Please change `out` to a return type instead of `void`. Variable name can be just `ret`